Sources/UIScrollView.swift

```diff
@@ -120,11 +158,27 @@ open class UIScrollView: UIView {
     open var contentInset: UIEdgeInsets = .zero
     open var contentSize: CGSize = .zero
 
-
     // MARK: Scroll Indicators
 
+    // Matching Apple's value
     let indicatorThickness: CGFloat = 2.5
 
+    // Determined experimentally to be as close as possible to Apple's (likely exactly matching)
+    private let baseScrollIndicatorInsets = UIEdgeInsets(top: 2.5, left: 2.5, bottom: 2.5, right: 2.5)
+
+    public var indicatorStyle: IndicatorStyle = .`default` {
+        didSet { applyScrollIndicatorsStyle() }
```

Comment:
just making sure `applyScrollIndicatorsStyle` is called elsewhere (as well) - `didSet`s don't get called on `init`. If this is the case we should add a comment to that effect here too

Reply:
yes, it is being called on `init` separately. Not sure if I understand you correctly, should I add a comment stating that fact?